web/src/app/dialogs/FormDialog.js

```diff
@@ -148,12 +148,15 @@ function FormDialog(props) {
   const fs = fullScreen || (!isWideScreen && !confirm)
   return (
     <Dialog
-      disableBackdropClick={!isWideScreen || alert}
       fullScreen={fs}
       maxWidth={maxWidth}
       fullWidth
       open={open}
-      onClose={handleOnClose}
+      onClose={(event, reason) => {
+        // disable backdrop click
+        if (reason !== 'backdropClick' || (isWideScreen && !alert))
+          handleOnClose()
```

Comment:
Bit of a style nit: let's invert this. If the precondition fails, short-circuit and exit without doing anything, otherwise, continue with the function.

It makes it clear we only intend to prevent closing for backdrop clicks on mobile or for alert dialogs.

```suggestion
        if (reason === 'backdropClick' && (!isWideScreen || alert)) {
          // disable backdrop for mobile and alerts
          return
        }

        handleOnClose()
```